base/data/Campaign.js

```diff
@@ -169,52 +169,35 @@ Campaign.hideAdverts = (topCampaign, campaigns) => {
  * Get all ads associated with the given campaigns
  * @param {Campaign} topCampaign 
  * @param {?Campaign[]} campaigns 
- * @returns {Advert[]}
+ * @param {?KStatus} status
+ * @returns PromiseValue(Advert[])
  */
-Campaign.fetchAds = (topCampaign, campaigns) => {
-    
-    let {
-        status,
-        'gl.status': glStatus
-    } = DataStore.getValue(['location', 'params']) || {};
-    if ( ! status) status = (glStatus || C.KStatus.PUB_OR_ARC);
+Campaign.fetchAds = (topCampaign, campaigns, status=KStatus.DRAFT) => {
```

Comment:
Could you switch this to named parameters? Thanks.

NB: If there is a matching PR in my-loop, please use identical branch names to align them.